docs/source/user_guide/config/data_settings.rst

```diff
@@ -98,6 +98,12 @@ Preprocessing
 - ``preload_weight (dict)`` : Has the format ``{k (str): v (float)}, ...``. ``k`` is a token field, representing the IDs of each row of preloaded weight matrix. ``v`` is a float-like field. Each pair of ``k`` and ``v`` should be from the same atomic file. This arg can be used to load pretrained vectors. Defaults to ``None``.
 - ``normalize_field (list)`` : List of filed names to be normalized. Note that only float-like fields can be normalized. Defaults to ``None``.
 - ``normalize_all (bool)`` : Normalize all the float like fields if ``True``. Defaults to ``None``.
+- ``discretization (dict)`` : Has the format ``{k (str): v (dict)}, ...``. ``k`` is a float field, representing the fields to be discretized. ``v`` is a config dict which have 2 keys: ``method`` and ``bucket``, which respectively control the discretization strategy.
```

Comment:
TO DO: add some docs for default parameters including DDP settings such as `ip`, `port`, `nproc` and `offset`.